mocks.ts

```diff
@@ -0,0 +1,46 @@
+export interface Car {
+    id: string;
+    brand: string;
+    model: string;
+    year: number;
+    description: string;
+    price: number;
+    inStock: boolean;
+    imgUrl: string;
+}
+
+export const cars: Car[] = [
+    {
+        id: '0',
+        brand: 'Audi',
+        model: 'A6',
+        year: 2005,
+        description: '',
+        price: 10000,
+        inStock: true,
+        imgUrl: 'https://content.onliner.by/automarket/generations/136/256x192/879d1732e52056f106ac684b9f4ce845.png',
+    },
+
+    {
+        id: '2',
+        brand: 'Audi',
+        model: 'A8',
+        year: 2005,
+        description: '',
+        price: 15000,
+        inStock: true,
+        imgUrl: 'https://content.onliner.by/automarket/generations/136/256x192/5915a25667a7bf84b63a51a383b57013.png',
+    },
+
+
+    {
+        id: '3',
+        brand: 'Audi',
+        model: 'A5',
+        year: 2005,
+        description: '',
+        price: 12000,
+        inStock: true,
```

Comment:
Interesting product)) There is no option to comment about ability to add count to car model) Each one is uniq...

Reply:
I will transform it to toy cars :)